src/frontend/src/handler/create_function.rs

```diff
@@ -293,25 +264,23 @@ pub async fn handle_create_function(
     Ok(PgResponse::empty_result(StatementType::CREATE_FUNCTION))
 }
 
-/// Upload wasm binary to object store.
-async fn upload_wasm_binary(
-    wasm_storage_url: &str,
-    object_name: &str,
-    wasm_binary: Bytes,
-) -> Result<()> {
-    // Note: it will panic if the url is invalid. We did a validation on meta startup.
```

Comment:
As the comment says `build_remote_object_store` will panic, so we should not use it directly in the current form..

Reply:
so could you recommend one that I should use here? 👀

Reply:
I think the easiest way is to only accept a HTTP url to download 🤡

Reply:
Otherwise maybe need some code to wrap opendal, or change how `object_store` handles error

Reply:
I'd like to only support local file system `fs://` first as our e2e tests depend on this. 🤡